.github/CODEOWNERS

```diff
@@ -1 +1,47 @@
-# To be done
+# Modules
+src/modules/automod/ @WilsontheWolf @KommentatorForAll
+src/modules/autoresponses/ @KommentatorForAll
+src/modules/cooldownNotifications/ @KommentatorForAll
+src/modules/dmCommands/ @KommentatorForAll
+src/modules/errorHandling/ @KommentatorForAll
+src/modules/inlineCommands/ @KommentatorForAll
+src/modules/interactions/ @WilsontheWolf @KommentatorForAll
+src/modules/leveling/ @KommentatorForAll
+src/modules/messagePreview/ @Chicken
+src/modules/modCommands/ @WilsontheWolf
+src/modules/startupModule/ @KommentatorForAll
+src/modules/threadjoin/ @Chicken
+src/modules/twitter/ @Chicken
+src/modules/welcome/ @Chicken
+
+# Constants
+src/constants/ @KommentatorForAll @WilsontheWolf @Chicken
+
+# DTO's
+src/dto/OffenceDto.js @WilsontheWolf
+src/dto/ResponseDto.js @KommentatorForAll
+src/dto/UserDto.js @WilsontheWolf @KommentatorForAll
+
+# Base Code
+src/loadModules.js @KommentatorForAll
+src/modUtil.js @WilsontheWolf
+src/notifcations.js @KommentatorForAll
+src/permissons.js @KommentatorForAll @WilsontheWolf
+src/roles.js @WilsontheWolf
+src/util.js @WilsontheWolf @KommentatorForAll
+src/xpHandling.js @KommentatorForAll
+src/index.js @WilsontheWolf @KommentatorForAll @Chicken
+
+# Config
+.env.example @Chicken
```

Comment:
Ain't this all of us too?
or at least me too, i mean, i committed almost half of the file

Reply:
Hmmh, true, imo just leave the .env file without owner, I still plan on revisiting the config system at somepoint.

Reply:
Yeah but chicken always yells at us about newlines